src/log.rs

```diff
@@ -0,0 +1,105 @@
+pub use log::Level;
+pub use log::LevelFilter;
+
+use crate::{LogLevel, Verbosity, VerbosityFilter};
+
+impl<L: LogLevel<LevelFilter = LevelFilter>> Verbosity<L> {
+    /// Get the log level.
+    ///
+    /// `None` means all output is disabled.
+    pub fn log_level(&self) -> Option<Level> {
+        self.log_level_filter().to_level()
+    }
```

Comment:
To move this up into the shared implementation (and the equivalent `tracing_level` method below), requires that we can make a generic type bound that converts the associated LevelFilter type to an associated Option<Level> type for both tracing and log crates. The log crate does not (yet) implement the necessary conversion methods. I've submitted a PR [to the log crate that should address that problem](https://github.com/rust-lang/log/pull/644) however. If that gets merged, then there is no type specific code differences on Verbosity left, and the entire implementation can work by just leveraging the conversion methods.

I think it would be reasonable to wait and see where that PR goes before completing this PR, as that would be a fairly neat outcome I think.